src/views/AppView.tsx

```diff
@@ -123,8 +125,12 @@ function AppView() {
       <div className="App-chart-n-rep-parent">
         <div className="chart-n-rep-child">
           <ResponsiveContainer width="50%" height="100%">
-            <Chart data={handleDataChange()} />
+            <Chart
+              data={handleDataChange()}
+              setCurrentTrans={setCurrentTrans}
```

Comment:
Same note as above, 
- Leave this out and replace with an onHover/onClick feature to display data info.